packages/react-components/src/components/styles/ReactionButton.styles.ts

```diff
@@ -28,8 +28,13 @@ export const playFrames = memoizeFunction(() =>
  *
  * @private
  */
-export const emojiStyles = (backgroundImage: string, animationPlayState: string): React.CSSProperties => {
+export const emojiStyles = (
+  backgroundImage: string,
+  animationPlayState: string,
+  frameCount?: number
```

Comment:
This should be required, we need the caller to always pass in the correct frame count for the sprite animation to work

Reply:
(same for reactionRenderingStyle in video tile styles)